packages/server/lib/controllers/onboarding.controller.ts

```diff
@@ -432,7 +432,12 @@ class OnboardingController {
             }
 
             logCtx = await logContextGetter.create(
-                { id: String(activityLogId), operation: { type: 'action' }, message: 'Start action' },
+                {
+                    id: String(activityLogId),
+                    operation: { type: 'action' },
+                    message: 'Start action',
+                    expiresAt: new Date(Date.now() + 86400 * 1000).toISOString()
+                },
```

Comment:
1 day for an action that's pretty generous

Reply:
it's just for the heartbeat, unfortunately I have no way to have something more precise. But if everything works well everything should be handled by temporal/v2 :D

Reply:
Would be nice to abstract out these times to one place in case we need to adjust

Reply:
yeah good point, actually I wasn't aware of the constant you shared on previous message so maybe I can expose them globally somehow